File: go/devnest/helloUser.go
// You are given a String str. Return the String by Concatenating it with String "Hello, " and an exclamation mark at the end “!”.

// Input Format
// First Parameter - String str

// Output Format
// Return a String

package main

import (
	"fmt"
)

func solve(str string) string {
	res := "Hello " + str + "!"
	return res
}

func main() {
	var str string
	fmt.Print("Enter your name: ")
	fmt.Scanf("%s", &str)
	res := solve(str)
	fmt.Println(res)
}

File: python/devnest/helloUser.py
# You are given a String str. Return the String by Concatenating it with String "Hello, " and an exclamation mark at the end “!”.

# Input Format
# First Parameter - String str

# Output Format
# Return a String

def solve(str):
    # CODE HERE
    return ("Hello, " + str + "!")


if __name__ == "__main__":
    str = input("Please enter your name\n")
    print(solve(str))
